omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/field/LocationsFragmentController.java

```diff
@@ -0,0 +1,39 @@
+package org.openmrs.module.locationbasedaccess.fragment.controller.field;
+
+import org.openmrs.Location;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.appui.UiSessionContext;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Fields and actions to provide registrationInfo view.
+ *
+ * @author Romain Buisson
+ */
+public class LocationsFragmentController {
```

Comment:
What are we using this for?

Reply:
I used this to add the system all locations to the Location Selection fragment - Combo box.  And current session location will be automatically selected as this location value in the fragment combo box.